src/Microsoft.PowerShell.TextUtility.psd1

```diff
@@ -13,14 +13,14 @@
     PowerShellVersion = '5.1'
     FormatsToProcess = @('Microsoft.PowerShell.TextUtility.format.ps1xml')
     CmdletsToExport = @(
-        'Compare-Text','ConvertFrom-Base64','ConvertTo-Base64'
+        'Compare-Text','ConvertFrom-Base64','ConvertTo-Base64',"Convert-TextTable"
     )
     PrivateData = @{
         PSData = @{
             LicenseUri = 'https://github.com/PowerShell/TextUtility/blob/main/LICENSE'
             ProjectUri = 'https://github.com/PowerShell/TextUtility'
             ReleaseNotes = 'Initial release'
-            Prerelease = 'Preview1'
+            Prerelease = 'Preview2'
```

Comment:
I wonder if we should just leave this as `Preview` since we're updating the version anyways

Reply:
sure - I'll do that

Reply:
Can we please remove the prerelease label? This module hasn't reached 1.0 to indicate stable yet. Not having any stable versions of a package is a hassle to install since `-Prerelease` has to be used. The main benefit to having prerelease in my opinion is when a stable version already exists.